aspnetcore/fundamentals/host/web-host.md

```diff
@@ -40,7 +40,7 @@ public class Program
 * Loads [host configuration](#host-configuration-values) from:
   * Environment variables prefixed with `ASPNETCORE_` (for example, `ASPNETCORE_ENVIRONMENT`).
   * Command-line arguments.
-* Loads app configuration from:
+* Loads app configuration in the following order from:
```

Comment:
List Azure Key Vault here too. Consider using the term "Secret Manager" in the User secrets bullet.

Reply:
> List Azure Key Vault here too. 

This is the providers in the framework ... the hard-wired ones. AKV isn't there by default.

"Secret Manager" :+1: ... making that change now.